app/views/spree/admin/orders/_list.html.haml

```diff
@@ -0,0 +1,16 @@
+%table.payment-list
+  %thead
+    %tr
+      %th= "#{Spree.t('date')}/#{Spree.t('time')}"
+      %th.amount= t(:amount)
+      %th= t(:payment_method)
+      %th.payment-state= t(:payment_state)
+  %tbody
+    - payments.each do |payment|
+      %tr
+        %td= pretty_time(payment.created_at)
+        %td.amount= payment.display_amount.to_html
+        %td
+          %span.payment-method-name= payment_method_name(payment)
+          %span.payment-method-description= payment.payment_method.description
+        %td.payment-state= t(payment.state, scope: :payment_states, default: payment.state.capitalize)
```

Comment:
there's a missing empty line.
in this case, because it's a list of standard payment states I think lazylookup can be skipped.